cuda_core/tests/test_module.py

```diff
@@ -248,6 +258,118 @@ def test_num_args_error_handling(deinit_all_contexts_function, cuda12_prerequisi
         _ = krn.num_arguments
 
 
+@pytest.mark.parametrize("block_size", [32, 64, 96, 120, 128, 256])
+@pytest.mark.parametrize("smem_size_per_block", [0, 32, 4096])
+def test_saxpy_occupancy_max_active_block_per_multiprocessor(get_saxpy_kernel, block_size, smem_size_per_block):
```

Comment:
Q: Do we need to mention `saxpy` in every test name? Does it matter what kernel we use in tests?